spring/src/main/java/io/seata/spring/tcc/TccActionInterceptor.java

```diff
@@ -39,7 +40,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(TccActionInterceptor.class);
 
-    private static final String DUBBO_PROXY_NAME_PREFIX="com.alibaba.dubbo.common.bytecode.proxy";
+    private static final String DUBBO_PROXY_NAME_PREFIX = "com.alibaba.dubbo.common.bytecode.proxy";
```

Comment:
Do you need to consider apache dubbo version?

Reply:
see TransactionPropagationFilter.java